LPS-93185 Use current site and user

darquesdev opened the pull request.

stiemannkj1 committed:
LRQA-33560 Update TCK theme for compatability with Liferay master

stiemannkj1 committed:
LRQA-33560 Update TCK portlet bridge for compatability with the Portlet 3.0 TCK

stiemannkj1 committed:
LRQA-33560 Update TCK ant scripts for compatability with the Portlet 3.0 TCK

stiemannkj1 committed:
LRQA-33560 add author

stiemannkj1 committed:
LRQA-33560 remove dead code

stiemannkj1 committed:
LRQA-33560 add htmlunit-tests.xml file and determine (non-htmlunit) tck-tests.xml from it

stiemannkj1 committed:
LRQA-33560 Add all portlets to test pages

stiemannkj1 committed:
LRQA-33560 Add/fix TCK test case config files

stiemannkj1 committed:
LRQA-33560 Split build-test-tck.xml tck-jdk8 target into 3 different standalone targets

stiemannkj1 committed:
LRQA-33560 Update portals-pluto commit

stiemannkj1 committed:
LRQA-33560 don't fail build when tests fail

stiemannkj1 committed:
LRQA-33560 clean up patch after run

stiemannkj1 committed:
LRQA-33560 remove old TCK set up

stiemannkj1 committed:
LRQA-33560 fix webdriver dir

stiemannkj1 committed:
LRQA-33560 fix ant scripts to run correctly in CI

stiemannkj1 committed:
LRQA-33560 set up app server with 1g MaxMetaspace and SSL

stiemannkj1 committed:
LRQA-33560 fix TCK properties

stiemannkj1 committed:
LRQA-33560 use Pluto version with reduced logging and skippable tests

stiemannkj1 committed:
LRQA-33560 allow prepare-tck-app-server target to be run on it's own

stiemannkj1 committed:
LRQA-33560 include TestModule tck tests

stiemannkj1 committed:
LRQA-33560 include TestModule tck tests as non-exclusive mode tests

stiemannkj1 committed:
LRQA-33560 update pluto

stiemannkj1 committed:
LRQA-33560 run V3RenderStateTests in chrome non-exclusive

stiemannkj1 committed:
LRQA-33560 Ignore failing tests

vicnate5 committed:
LRQA-33560 Not needed as these are already blacklisted by our testing framework

Commit:
LPS-93114 Fragment Renderer and tracker

vicnate5 committed:
LRQA-33560 Enable tests that are no longer failing

vicnate5 committed:
LRQA-33560 Add tickets for failing tests

jrao committed:
LPS-92284 Don't show View in Context link for blog entries with no suitable context (i.e., don't display View in Context links that would result in a 404)

drewbrokke committed:
LPS-90255 portal-security-sso-ntlm-api - makes NtlmConfiguration system-scoped so that its entry does not show up in Instance Settings

drewbrokke committed:
LPS-90255 configuration-admin-web - does not show a configuration screen entry inside its category if it's not at the proper scope

drewbrokke committed:
LPS-90255 portal-settings-web - tracks instances of PortalSettingsConfigurationScreenContributor and dynamically registers a ConfigurationScreen entry for each one

drewbrokke committed:
LPS-90255 portal-settings-api - adds PortalSettingsConfigurationScreenContributor interface to unify adding ConfigurationScreens to the new Instance Settings

drewbrokke committed:
LPS-90255 portal-security-sso-ntlm-settings-authentication-web - registers PortalSettingsFormContributor in place of the dynamic include

drewbrokke committed:
LPS-90255 portal-security-sso-cas-api - CASConfiguration - render auto-generated UI only at System scope

drewbrokke committed:
LPS-90255 portal-settings-authentication-cas-web - registers form as a ConfigurationScreen

vicnate5 committed:
LRQA-33560 Fix theme build for changes at head

drewbrokke committed:
LPS-90255 portal-settings-web - if a redirect is present on error, send redirect

Commit:
LPS-93114 Add rendererType and rendererKey to FragmentEntryLink

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

darquesdev commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/3281/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-8.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3281/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 21 out of 24 jobs passed in 1 hour 14 minutes 39 seconds 484 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/e4ca8e0fc87b963340338fc413fe068ad88761dc">e4ca8e0fc87b963340338fc413fe068ad88761dc</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/b812a7263e872687cdc6ba39be44651326c19358">b812a7263e872687cdc6ba39be44651326c19358</a></p>21 out of 24 jobs PASSED<div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/3281/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/41683/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/199216/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</strong></strike></a></li></ul></div><details><summary><strong>21 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/266468/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/41682/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/198923/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/266469/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/267422/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/267423/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/294167/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/199215/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/267424/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/294168/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/198924/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/266470/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/257771/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/267425/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/266471/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/199217/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/294169/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/198925/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/280180/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/280181/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/155477/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-8.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3281/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/199216/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/199216//consoleText">AXIS_VARIABLE=0,label_exp=!master #199216</a><pre><code>     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:layout:layout-content-page-editor-web:npmRunTest'.
     [exec] &gt; java.io.IOException: Process '[/opt/dev/projects/github/liferay-portal/build/node/bin/node, /opt/dev/projects/github/liferay-portal/modules/yarn-1.13.0.js, run, test]' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:layout:layout-content-page-editor-web:npmRunTest'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [exec] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [exec] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [exec] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [exec] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)</code></pre></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">acceptance upstream results</a> at e4ca8e0fc87b963340338fc413fe068ad88761dc:</strong></summary><ol><li><div><h5><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/41683/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>17 Tests Passed.<br/>3 Tests Failed.</p></div><ol><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/41683//consoleText">AXIS_VARIABLE=1,label_exp=!master #41683</a><ol><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/41683//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DisplayPagesWithStagingAndWebContent_PublishDisplayPageMarkedAsDefault_">LocalFile.DisplayPagesWithStagingAndWebContent#PublishDisplayPageMarkedAsDefault</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1554205518517/test-portal-acceptance-pullrequest(master)/3281/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.DisplayPagesWithStagingAndWebContent_PublishDisplayPageMarkedAsDefault/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1554205518517/test-portal-acceptance-pullrequest(master)/3281/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.DisplayPagesWithStagingAndWebContent_PublishDisplayPageMarkedAsDefault/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1554205518517/test-portal-acceptance-pullrequest(master)/3281/functional-tomcat90-mysql57-jdk8/0/1/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/41683//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DisplayPagesWithStagingAndWebContent_PublishEditedDisplayPage_">LocalFile.DisplayPagesWithStagingAndWebContent#PublishEditedDisplayPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1554205518517/test-portal-acceptance-pullrequest(master)/3281/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.DisplayPagesWithStagingAndWebContent_PublishEditedDisplayPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1554205518517/test-portal-acceptance-pullrequest(master)/3281/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.DisplayPagesWithStagingAndWebContent_PublishEditedDisplayPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1554205518517/test-portal-acceptance-pullrequest(master)/3281/functional-tomcat90-mysql57-jdk8/0/1/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/41683//consoleText">AXIS_VARIABLE=2,label_exp=!master #41683</a><ol><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/41683//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DisplayPagesWithWebContent_ViewDisplayPageViaAssetPublisher_">LocalFile.DisplayPagesWithWebContent#ViewDisplayPageViaAssetPublisher</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1554205518517/test-portal-acceptance-pullrequest(master)/3281/functional-tomcat90-mysql57-jdk8/0/2/LocalFile.DisplayPagesWithWebContent_ViewDisplayPageViaAssetPublisher/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1554205518517/test-portal-acceptance-pullrequest(master)/3281/functional-tomcat90-mysql57-jdk8/0/2/LocalFile.DisplayPagesWithWebContent_ViewDisplayPageViaAssetPublisher/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1554205518517/test-portal-acceptance-pullrequest(master)/3281/functional-tomcat90-mysql57-jdk8/0/2/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol></details></details></html>

andresfulla commented:
👍

Commit:
LPS-93048 Add data layout permission into rest open api yaml file

marcelabc committed:
LPS-93221 Add 'customProperties' property in DataDefinitionField

marcelabc committed:
LPS-93221 buildRest

marcelabc committed:
LPS-93221 create new util methods

marcelabc committed:
LPS-93221 Move CaptchaFieldType to rest resource

Commit:
LPS-93226 Modifying delete resourceImpl

Commit:
LPS-93048 REST api generated files

Commit:
LPS-93226 Remove ignore from get and put tests

Commit:
LPS-93048 Add data layout permission resource implementation

vicnate5 committed:
LRQA-33560 Do not use custom theme

kenjiheigel committed:
LRCI-119 Clean up exception handling while getting upstream failure data

shuyangzhou committed:
LRQA-33560 Remove tck theme

Commit:
LPS-93226 Adjust data definition permission tests

brianchandotcom committed:
LPS-91856 Reorder

Commit:
LPS-93078 users-admin-web - Adjust personal menu entry order

Commit:
LPS-93078 product-navigation - Move site selector to personal menu

shuyangzhou committed:
LRQA-33560 SF

shuyangzhou committed:
LRQA-33560 Add a space to protect from repeated appending.

shuyangzhou committed:
LRQA-33560 Wrong key, we are defaulting to 15 now.

Commit:
LPS-93078 product-navigation-site-administration - Hide Site Administration panel category when irrelevant

shuyangzhou committed:
LRQA-33560 Redundant setting, web.xm always wins anyway.

shuyangzhou committed:
LRQA-33560 Just did this at the beginning of the target

shuyangzhou committed:
LRQA-33560 Deploy to the right folder

shuyangzhou committed:
LRQA-33560 Switch to config

shuyangzhou committed:
LRQA-33560 I don't see anyone reading this.

shuyangzhou committed:
LRQA-33560 Config logger the normal way

shuyangzhou committed:
LRQA-33560 Not used

shuyangzhou committed:
LRQA-33560 Auto SF

shuyangzhou committed:
LRQA-33560 Manual SF

kenjiheigel committed:
LRCI-119 Store variable to denote if upstream comparison is not available

kenjiheigel committed:
LRCI-119 Update the github message

Commit:
LRQA-47900 Add macro to go to configuration on instance setting

Commit:
LRQA-47900 Add steps to click ellipsis on CAS

Commit:
LRQA-47900 Update the case to more reasonable

pavel-savinov committed:
LPS-93181 Get appropriate class type ids from asset list entry properties

pavel-savinov committed:
LPS-93181 Add util for asset helper to be able to use it from model implementations

pavel-savinov committed:
LPS-93181 Use asset helper to get asset entries using indexer instead of the database

Commit:
LPS-93048 SF

Commit:
LPS-93048 Rename parameter data layout id for permission

Commit:
LPS-93048 REST api generated files

Commit:
LRQA-47900 Use form fields

jrao committed:
LPS-91413 Use groupId from _fileEntry not from themeDisplay

jrao committed:
LPS-91413 Update SF exception list

Commit:
LPS-93078 product-navigation-product-menu-web - Hide product menu when there are no entries

Commit:
LPS-93078 product-navigation-product-menu-web - Impersonation options should no longer show in the product menu

marcelabc committed:
LPS-93221 Add dependencies

Commit:
LPS-93078 product-navigation-taglib - Hide control menu entirely when there are no entries

Commit:
LPS-93078 product-navigation-product-menu-theme-contributor - Fix template css when the product menu should be hidden

ChrisKian committed:
LPS-91413 Keep existing API and deprecate

Commit:
LPS-93226 Add call to addStructure from TestHelper

shuyangzhou committed:
LRQA-33560 Remove dead code

shuyangzhou committed:
LRQA-33560 Always 1, never changes.

shuyangzhou committed:
LRQA-33560 Don't create page unless there is portlet.

kenjiheigel committed:
LRCI-119 Reword

shuyangzhou committed:
LRQA-33560 We just want the default company, just ask for it.

shuyangzhou committed:
LRQA-33560 We just want the default admin user, just ask for it.

shuyangzhou committed:
LRQA-33560 Not thrown

shuyangzhou committed:
LRQA-33560 Apply

shuyangzhou committed:
LRQA-33560 Prevent permission checker threadlocal leaking.

shuyangzhou committed:
LRQA-33560 Use default admin user, rather than default user.

shuyangzhou committed:
LRQA-33560 Avoid deprecated api usage.

shuyangzhou committed:
LRQA-33560 Configure modules' logging once they reach to RESLOVED state, to make sure loggers are usable during the bundle activation process.

shuyangzhou committed:
LRQA-33560 Not necessary, default metaspace is big enough

ealonso committed:
LPS-93359 Removes small image from table view

ealonso committed:
LPS-93359 Implement vertical card for journal ddm templates

ealonso committed:
LPS-93359 Creates new vertical card views

ealonso committed:
LPS-93359 Moves actions to a java class

adolfopa committed:
LPS-91413 This method was new, no need to deprecate

Commit:
LPS-93114 Generated files (service builder)

brianchandotcom committed:
LPS-93114 regen

Commit:
LPS-91856 Move constants and logic from FragmentEntryTypeConstants into FragmentConstants

Commit:
LPS-91856 Move constants and logic from FragmentEntryTypeConstants into FragmentConstants -- Change Usages

ealonso committed:
LPS-93434 New contributor for small image in journal

ealonso committed:
LPS-93434 New contributor for small image in blogs entry

ealonso committed:
LPS-93434 Moves getThemeDisplay to a Base class to avoid code duplication

ealonso committed:
LPS-93434 Show images only if we have one

adolfopa committed:
LPS-92284 Add central registry for PortletLayoutFinders

adolfopa committed:
LPS-92284 Use central registry in Blog and DM

adolfopa committed:
LPS-92284 Remove methods from the API

This whole "View in Context" thing will probably change quite a lot with all the
ongoing work on Display Pages.  Don't expose this on the API until we're sure
we'll need this in the future.

These are the only two componentes that use LayoutFinders, so this duplication
won't propate to other places.

adolfopa committed:
LPS-92284 Semver

Commit:
LPS-93114 Add .lfrbuild for the two new modules

ealonso committed:
LPS-91872 Don't show edit icon on edit mode

ealonso committed:
LPS-91872 Uses an icon pencil instead of a toggle

ealonso committed:
LPS-91872 Rename

adolfopa committed:
LPS-92209 Remove namespace

The postForm function already prepends the namespace.

adolfopa committed:
LPS-92209 Choose right editor for the current format

Commit:
LPS-93095 Prepare service

Commit:
LPS-93095 Update service implementation

Commit:
LPS-93095 Regen

kenjiheigel committed:
LRCI-120 Timeout dist task after 2 hours

Commit:
LPS-93095 Support staging for segments

Commit:
LPS-93095 Add test

shuyangzhou committed:
LRQA-33560 SF

shuyangzhou committed:
LRQA-33560 Inline

shuyangzhou committed:
LRQA-33560 Simplify

shuyangzhou committed:
LRQA-33560 What's this for? Not needed at all.

shuyangzhou committed:
LRQA-33560 Use TCK site group as an indicator to avoid repeatedly trying to recreate the site and pages.

shuyangzhou committed:
LRQA-33560 SF

shuyangzhou committed:
LRQA-33560 Inline

shuyangzhou committed:
LRQA-33560 Newly created Layouts, nothing to reset

shuyangzhou committed:
LRQA-33560 Inline

shuyangzhou committed:
LRQA-33560 Not used

adolfopa committed:
LPS-92284 Register finder for both models

shuyangzhou committed:
LRQA-33560 Useless model class

shuyangzhou committed:
LRQA-33560 Not used

Commit:
LPS-93048 Fix content space id parameter

shuyangzhou committed:
LRQA-33560 Portlets need to be added to the page regardless of the source bundle states.

Commit:
LPS-93048 Regenerate rest

marcelabc committed:
LPS-93221 Move FieldType to rest resource

Commit:
LPS-93048 Fix permission constant

Commit:
LPS-93048 Extract permission checking and persistence

brianchandotcom committed:
LPS-92284 SF

shuyangzhou committed:
LRQA-33560 These never match anything.

shuyangzhou committed:
LRQA-33560 Inline

Commit:
LPS-93048 Extract permission model persistence and apply

victorg1991 committed:
LPS-93270 Add patch and required for StructuredContentFolder

victorg1991 committed:
LPS-93270 Autogenerated

victorg1991 committed:
LPS-93270 Update tests

victorg1991 committed:
LPS-93270 Add exception mappers. I've used hardcoded messages because the exception message was not useful enough

victorg1991 committed:
LPS-93270 Rename to follow same pattern

shuyangzhou committed:
LRQA-33560 Simplify regex matching

shuyangzhou committed:
LRQA-33560 Pass around portletIds directly

shuyangzhou committed:
LRQA-33560 Get PermissionChecker only when it is needed, to avoid unnecesary ThreadLocal getting.

shuyangzhou committed:
LRQA-33560 Skip permission checking completely, it is admin user anyway.

pei-jung committed:
LPS-93078 (product-navigation-control-menu-theme-contributor) fix template css when control menu is hidden

shuyangzhou committed:
LRQA-33560 The scope group id is never used.

shuyangzhou committed:
LRQA-33560 Remove useless logging

brianchandotcom committed:
LPS-93181 Only used in one place

brianchandotcom committed:
LPS-93181 sort

pyoo47 committed:
LRCI-130 Standardize equals() implementations

brianchandotcom committed:
LPS-93181 sF

brianchandotcom committed:
LPS-93181 More accurate?

shuyangzhou committed:
LRQA-33560 Inline

shuyangzhou committed:
LRQA-33560 SF

brianchandotcom committed:
LPS-93181 Sort

brianchandotcom committed:
LPS-93181 Typo

shuyangzhou committed:
LRQA-33560 Just read the file

pei-jung committed:
LPS-93078 Fix person menu not accessible after clicking on an item

shuyangzhou committed:
LRQA-33560 Simplify XML parsing

shuyangzhou committed:
LRQA-33560 Not used

brianchandotcom committed:
LPS-90255 Please fix the request attribute with a string

shuyangzhou committed:
LRQA-33560 Just need properties

drewbrokke committed:
LPS-90255 portal-settings-web - uses constants for attributes

shuyangzhou committed:
LRQA-33560 Inline

shuyangzhou committed:
LRQA-33560 Properly declare dependencies

shuyangzhou committed:
LRQA-33560 Specify the config file directly, and make it required.

brianchandotcom committed:
LPS-93270 So I can see it break if that method is not generated in the future

brianchandotcom committed:
LPS-93270 wordsmith

petershin committed:
LPS-89415 Move operationId check to another method (no logic changes)

brianchandotcom committed:
LPS-93226 SF

petershin committed:
LPS-89415 Sort methods alphabetically

petershin committed:
LPS-89415 Camel case path parameter names

drewbrokke committed:
LPS-90255 portal-settings-web - rename the ServiceTrackerCustomizer to simply be a Tracker class that constructs an anonymous ServiceTrackerCustomizer inline

brianchandotcom committed:
LPS-93078 Consistency

brianchandotcom committed:
LPS-93078 Honor method name

petershin committed:
LPS-89415 Generated

brianchandotcom committed:
LRCI-119 good enough

brianchandotcom committed:
LRCI-130 prep next

shuyangzhou committed:
LRQA-33560 Deploy tck bridge before starting portal, and stop using log message waiting.

shuyangzhou committed:
LRQA-33560 SF

shuyangzhou committed:
LRQA-33560 Move the marco back to where it was.

shuyangzhou committed:
LRQA-33560 Not used anymore

kyle-miho committed:
LPS-92010 Re-enable failing test

michaelhashimoto committed:
LRCI-64 Fix applying documentum properties to match new lpkg folder structure

brianchiu committed:
LPS-91872 Update tests to use pencil

pei-jung committed:
LPS-93433 Fix PortletTitleTest for PersonalMenuPortlet

brianchandotcom committed:
LPS-93048 regen

brianchandotcom committed:
LPS-93048 feels more right

brianchandotcom committed:
LPS-93048 auto SF

brianchandotcom committed:
LPS-93226 match usage in headless-delivery/headless-delivery-test/src/testIntegration/resources/com/liferay/headless/delivery/resource/v1_0/test/dependencies/test-structured-content-structure.json

brianchandotcom committed:
LPS-93226 Simplify

brianchandotcom committed:
LPS-93226 Not used

brianchandotcom committed:
LPS-93226 Only internal for now

brianchandotcom committed:
LPS-93226 auto SF

brianchandotcom committed:
LPS-93226 Add @Ignore back, if you override the test method, you need to at least call it so we know all of the default tests are run (it breaks right now, I need your help to fix it)

brianchandotcom committed:
LPS-93226 Expose 'userNameAndPassword', set this var before calling invoke*

brianchandotcom committed:
LPS-93226 regen

brianchandotcom committed:
LPS-93221 Match src/main/java/com/liferay/data/engine/rest/internal/rule/function/v1_0/

brianchandotcom committed:
LPS-93221 Like this

brianchandotcom committed:
LPS-93221 Sort

brianchandotcom committed:
LPS-93221 Private it

brianchandotcom committed:
LPS-93221 Inline it

brianchandotcom committed:
LPS-93221 As used

brianchandotcom committed:
LPS-93221 Auto SF

brianchandotcom committed:
LPS-93221 This is fine

brianchandotcom committed:
LPS-93221 Use JSONUtil like in src/main/java/com/liferay/data/engine/rest/internal/dto/v1_0/util/*

brianchandotcom committed:
LPS-93221 Remove useless method setLocalizedProperty. Why are we transforming LocaleValue[] to a Map to a JSONObject? Please review code carefully.

brianchandotcom committed:
LPS-93221 Optimization... this one is harder to know of

brianchandotcom committed:
LPS-93221 Ah, just inline it

brianchandotcom committed:
LPS-93221 Simplify

brianchandotcom committed:
LPS-93221 Sort

brianchandotcom committed:
LPS-93221 Sort

brianchandotcom committed:
LPS-93221 Please look at the class you're modifying before adding new methods.. call the existing method

brianchandotcom committed:
LPS-93221 no need for your new method

brianchandotcom committed:
LPS-93221 this is faster, so I'm not making a throw away list

brianchandotcom committed:
LPS-93221 this is faster, so I'm not making a throw away map

brianchandotcom committed:
LPS-93221 This is OK

brianchandotcom committed:
LPS-93221 This time, we use the setters so that the constructor does not set itself "key = key" may do nothing

brianchandotcom committed:
LPS-93221 clean up some old code that didn't match

brianchandotcom committed:
LPS-93221 Can I do this, or will it break stuff? If this is bad, please revert. But I did a quick look and I think you always assume a value in there

brianchandotcom committed:
LPS-93221 auto SF

ealonso committed:
LPS-93357 Adds new property to allow users decided to enabled or not the subscriptions

shuyangzhou committed:
LRQA-33560 Auto SF

brianchandotcom committed:
LRQA-33560 Belongs here

brianchandotcom committed:
LRQA-33560 Rename

brianchandotcom committed:
LRQA-33560 Rename

brianchandotcom committed:
LRQA-33560 Rename

brianchandotcom committed:
LRQA-33560 SF

brianchandotcom committed:
LRQA-33560 SF

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as e7269d27c6407d6cfad8de03a73796fc3dddb831

jkappler committed:
LPS-92553 Fix dependency
